libtock/analog_comparator.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
```

Comment:
General comments on this file:

 - While C doesn't require a variable name for prototype declarations, they are helpful, please add them
 - We generally try to err on the side of explicit / verbose / clear rather than terse. Please change all the `ac_` to `analog_comparator_`